recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: dceb15fef556a2775121bcc695561df4ff0e09248e0ce3a2d58f5244a9f61421
 
 build:
-    number: 0
+    number: 1
```

Comment:
No package is overwriten @scopatz. The script `upload_or_check_non_existence` ensures that.
Bumping build number is recommended when there are functional changes, or metadata changes, like license, etc. Anyways, you added yourself as a maintainer, so that can be considered a metadata change.

Note that the only difference between `pytables-3.3.0-np111py35_0.tar.bz2` and `pytables-3.3.0-np111py35_1.tar.bz2` will be your name as the maintainer, nothing else.